python/cudf/cudf/tests/test_decimal.py

```diff
@@ -41,3 +49,114 @@ def test_from_arrow_max_precision():
         DecimalColumn.from_arrow(
             pa.array([1, 2, 3], type=pa.decimal128(scale=0, precision=19))
         )
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        cudf.Series(
```

Comment:
No test case to convert Int to Decimal types.

Reply:
So this PR is only for floats. I made it so Int conversions raise a notimplemented error